packages/synapse-react-client/src/components/QueryContext/QueryContext.tsx

```diff
@@ -44,52 +28,52 @@ export type QueryContextType<
   removeQueryFilter: ImmutableTableQueryResult['removeQueryFilter']
   /** Removes a value from a QueryFilter. If no more values remain in the filter, the filter is also removed */
   removeValueFromQueryFilter: ImmutableTableQueryResult['removeValueFromQueryFilter']
-  /** The error returned by the query request, if one is encountered */
-  error: SynapseClientError | null
-  /** The status of the asynchronous job. */
-  asyncJobStatus?: AsynchronousJobStatus<QueryBundleRequest, QueryResultBundle>
```

Comment:
Removed a central `asyncJobStatus`, since each component manages its own query requests. Individual components can get this and include it in their loading state, if desired.